fossology/fossology
# test(models): Added unit cases for Permissions, LicenseCandidate, Gro…

## Description
 This pull request introduces more unit test coverage for the models like Permissions, LicenseCandidate, Group, Decider,  FileInfo and Agent as reported in issue [#2755](https://github.com/fossology/fossology/issues/2773)

### Changes
**Enhanced existing test cases:**  Worked on the  current test suite to improve the coverage.
**Added new test cases:** Introduced additional tests to  validate all methods and member variables.

### How to Test ?

- Navigate to the ui_tests/api/models directory.
- Execute each test case individually.
- Verify that all tests pass without any errors or failures.

CC @shaheemazmalmmd @soham4abc @dushimsam @GMishx
